core/row/row.pl.sdoc

```diff
@@ -97,7 +97,23 @@ defoperator row_sort => q{
     '--buffer-size='      . conf 'row/sort-buffer',
     '--parallel='         . conf 'row/sort-parallel'), @_};
 
-defshort '/g', pmap q{row_sort_op        sort_args @$_}, sortspec;
+defoperator partial_sort => q{
+  my $sort_size = shift;
+  my @buff = ();
+  while (<STDIN>) {
+    push @buff, $_;
+    if (@buff == $sort_size) {
+      print join "", sort(@buff);
```

Comment:
You can drop `join ""` here; `print` will take multiple arguments and concat them all into the stdout buffer. (Not sure if it's always faster, but I think it usually is)